src/modules/shared/pages/SelectModule.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="row q-col-gutter-xl q-pa-lg flex items-center module-selector bg-grey-8">
+    <img src="../../../assets/logos/kriptone-logo.svg" alt="" class="logo" />
+    <div class="col-6">
+      <q-card class="card-selection" @click="vulnerabilityHandler">
+        <q-card-title> Kriptone Tools </q-card-title>
+      </q-card>
+    </div>
+    <div class="col-6">
+      <q-card class="card-selection border" @click="auditsHandler">
+        <q-card-title> Audits </q-card-title>
+      </q-card>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+  import { useQuasar } from 'quasar';
+  import { AUTH_TOKEN_NAMES } from 'src/constants/fusion-auth.constants';
+  import { computed } from 'vue';
+  import { useRouter } from 'vue-router';
+  import { VULNERABILITY_ROUTES } from 'vulnerability/routes/route-names';
+
+  const $q = useQuasar();
+  const router = useRouter();
+  const accessAudits = computed(() => sessionStorage.getItem(AUTH_TOKEN_NAMES.AUDITS) || false);
+
+  async function vulnerabilityHandler() {
+    await router.push({ name: VULNERABILITY_ROUTES.home.name });
+  }
+
+  function auditsHandler() {
+    if (accessAudits.value !== 'undefined') {
+      console.info('AUDITS MODULE');
+    } else {
+      $q.dialog({
+        html: true,
+        message:
+          'Para acceder al modulo de auditoria debe contactarse con <a href="url">test@kriptone.com</a>',
```

Comment:
`[Nit]` Spelling mistake


```suggestion
          'Para acceder al módulo de auditoría debe contactarse con <a href="url">test@kriptome.com</a>',
```